src/main/java/teammates/common/datatransfer/attributes/FeedbackQuestionAttributes.java

```diff
@@ -323,96 +323,97 @@ public int hashCode() {
 
     @Override
     public boolean equals(Object obj) {
-        if (this == obj) {
+        //+1 Complexity
+        if (this == obj) { //+1 Complexity
             return true;
         }
 
-        if (obj == null) {
+        if (obj == null) { //+1 Complexity
             return false;
         }
 
-        if (getClass() != obj.getClass()) {
+        if (getClass() != obj.getClass()) { //+1 Complexity
             return false;
         }
 
         FeedbackQuestionAttributes other = (FeedbackQuestionAttributes) obj;
 
-        if (courseId == null) {
-            if (other.courseId != null) {
+        if (courseId == null) { //+1 Complexity
+            if (other.courseId != null) { //+1 Complexity
                 return false;
             }
-        } else if (!courseId.equals(other.courseId)) {
+        } else if (!courseId.equals(other.courseId)) { //+1 Complexity
             return false;
         }
```

Comment:
For refactoring later, we could use the method [Objects.equals(object, object)](https://stackoverflow.com/questions/11271554/compare-two-objects-in-java-with-possible-null-values) as it will return true if both objects are null